db/schema.rb

```diff
@@ -122,8 +122,8 @@
   end
 
   create_table "topics", force: :cascade do |t|
-    t.string "name"
-    t.string "image"
+    t.string "label"
+    t.string "icon"
```

Comment:
Just asking, why do you need to change this? 
Anyway you have to create a migration to make this changes in the db, because we can't modify this file manually and I think that's why GitHub  actions is failing (maybe you only forgot to include the migration file in the pr)